src/WebJobs.Script/Host/ScriptInvocationArguments.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Script.Host
+{
+    public class ScriptInvocationArguments(IServiceProvider serviceProvider) : Dictionary<string, object>
```

Comment:
Can this also be internal?

Reply:
Yep. We can make that internal.

Reply:
Updated the visibility of `ScriptInvocationArguments` to internal.